pkg/pillar/types/locationconsts.go

```diff
@@ -13,7 +13,7 @@ const (
 	PersistDir = "/persist"
 	// PersistConfigDir is where we keep some configuration across reboots
 	PersistConfigDir = PersistDir + "/config"
-	// PersistStatusDir is where we keep some configuration across reboots
+	// PersistStatusDir is where we used to keep some configuration across reboots. Remove once upgradeconverter code is removed.
```

Comment:
I guess it should be a comment to the `PersistConfigDir` constant, not `PersistStatusDir`?

Reply:
Oops

Reply:
Fixed